frontend/src/pull-card/flags.tsx

```diff
@@ -32,6 +33,11 @@ function Flags({pull}: {pull: Pull}) {
          href={pull.linkToSignature(devBlock)}
          icon={faMinusCircle}
       />}
+      {draftBlock && <PullFlag
+         variant="draftBlock"
+         title={"Draft"}
+         icon={faMinusCircle}
```

Comment:
Is this supposed to be `faMinusSquare`? If not, you might want to drop the `faMinusSquare` above.

Reply:
I forgot to update the icon value, thank you!

Reply:
Resolved via https://github.com/iFixit/pulldasher/pull/329/commits/c3af2d9290bc2aa0cd95e35380937afc411fe2f0